packages/vue/src/components/molecules/SfGallery/SfGallery.vue

```diff
@@ -8,6 +8,7 @@
               v-for="(picture, index) in images"
               :key="'slide-' + index"
               class="glide__slide"
+              :style="{ width: `${imageWidth}px` }"
```

Comment:
@AdamPawlinski probably I fix these issues when I work on CSS loading order bug. However, we shouldn't change it by inline styles :) In this case this width should set glide :)  
https://github.com/DivanteLtd/storefront-ui/commit/b657e7ab16aa7981ac213816bc3105528967ba36
https://github.com/DivanteLtd/storefront-ui/commit/8382a41d7ad4ab151e20e0ec1528ed9f57b4fc88
https://github.com/DivanteLtd/storefront-ui/commit/c99938c98d7f7947443c5e8b7ea41ed33daa1d7a

Reply:
I know that, but I couldn't find other solution. I didn't find any in glide documentaion.